lib/SQL/Translator/Parser/DBIx/Class.pm

```diff
@@ -149,13 +149,16 @@ sub parse {
 
         $table->primary_key(@primary) if @primary;
 
-        my %unique_constraints = $source->unique_constraints;
-        foreach my $uniq (sort keys %unique_constraints) {
-            if (!$source->_compare_relationship_keys($unique_constraints{$uniq}, \@primary)) {
+        my $unique_constraints = $source->unique_constraints_info;
+        foreach my $uniq (sort keys %$unique_constraints) {
+            my $cons = $unique_constraints->{$uniq};
```

Comment:
*Lines get really long* is exclusively an aesthetic concern. On the other hand *author made the interpreter do extra work... but I can't see why* is more involved.

Either way - this was a minor nitpick as I was reading. I will fix it when I do the final review/rebase pass.

Ignore this particular comment.